ui/components/ChipGroup.tsx

```diff
@@ -28,11 +28,18 @@ function ChipGroup({ className, chips = [], onChipRemove, onClearAll }: Props) {
             <Chip
               label={isUndefined ? chip + "null" : chip}
               onDelete={() => onChipRemove([chip])}
+              className="filter-options-chip"
             />
           </Flex>
         );
       })}
-      {chips.length > 0 && <Chip label="Clear All" onDelete={onClearAll} />}
+      {chips.length > 0 && (
+        <Chip
+          className="filter-clearAll-Chip"
```

Comment:
Maybe rm this if not using. Just to keep things tidy, I think if we declare a className I'd assume we're using it.